scripts/system/create/entityProperties/html/entityProperties.html

```diff
@@ -8,45 +15,53 @@
-    
-</head>
-
-<body onload='loaded();'>
-    <div id="properties-list">
-        <div class='property container'>
-            <label id='placeholder-property-type'></label>
-            <div class='value'>
-                <div class='row flex-center' style='padding-bottom: 8px;'>
-                    <div id="placeholder-property-name" class="stretch"></div>
-                    <div id="placeholder-property-locked" class="shrink"></div>
-                    <div id="placeholder-property-visible" class="shrink"></div>
-                </div>
-                <div class='row'>
-                    <div id="placeholder-property-id" class="stretch"></div>
+    <head>
+        <title>Properties</title>
+        <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
+        <link rel="stylesheet" type="text/css" href="../../../html/css/edit-style.css">
+        <link rel="stylesheet" type="text/css" href="../../../html/css/colpick.css">
+        <link rel="stylesheet" type="text/css" href="../../../html/css/jsoneditor.css">
+        <link rel="stylesheet" type="text/css" href="tabs/tabs.css">    
+        <script type="text/javascript" src="qrc:///qtwebchannel/qwebchannel.js"></script>
+        <script src="../../../html/js/jquery-2.1.4.min.js"></script>
+        <script type="text/javascript" src="../../../html/js/colpick.js"></script>
+        <script type="text/javascript" src="../../../html/js/jsoneditor.min.js"></script>
+        <script type="text/javascript" src="../../../html/js/eventBridgeLoader.js"></script>
+        <script type="text/javascript" src="../../../html/js/spinButtons.js"></script>
+        <script type="text/javascript" src="../../../html/js/utils.js"></script>
+        <script type="text/javascript" src="../../../html/js/includes.js"></script>
+        <script type="text/javascript" src="js/underscore-min.js"></script>
+        <script type="text/javascript" src="js/createAppTooltip.js"></script>
+        <script type="text/javascript" src="js/draggableNumber.js"></script>
+        <script type="text/javascript" src="js/entityProperties.js"></script>    
+    </head>
+    <body onload='loaded();'>
+        <div id="properties-list">
+            <div class='property container'>
+                <label id='placeholder-property-type'></label>
+                <div class='value'>
+                    <div class='row flex-center' style='padding-bottom: 8px;'>
+                        <div id="placeholder-property-name" class="stretch"></div>
+                        <div id="placeholder-property-locked" class="shrink"></div>
+                        <div id="placeholder-property-visible" class="shrink"></div>
+                    </div>
+                    <div class='row'>
+                        <div id="placeholder-property-id" class="stretch"></div>
+                    </div>
                 </div>
             </div>
+            <table style="width:100%; min-height: 352px; display: block;">
```

Comment:
It would be better to not have inline styles. Instead, use classes and update scripts/system/html/css/edit-style.css similar to how other HTML in this file is styled.